### DISCUSSION TITLE
Is there a built in bool i can use that returns true when in a timeline?

### DISCUSSION THREAD

**Discussion by Theooolone** [dialogic-godot/dialogic#1729] [Q&A] [answered]
I'm implementing dialogic into my game to replace my own textbox system, is there a built in boolean I can use that returns true when inside a timeline? My character movement code depends on it to know to ignore inputs when in a textbox.

**Comment by Jowan-Spooner** [comment, marked as answer]
I would heavily suggest having some kind of state system in your game that has a "Dialogue" or some similar state. 

But what you are looking for is likely `Dialogic.has_active_layout_node()` which will likely only be true if you have a running dialog. 
Alternatively `Dialogic.current_timeline != null` will work very similarly.

**Reply by Theooolone** [reply]
This worked great, thank you!
